tests/src/PatternAssertion/DescriptionListAssert.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\PatternAssertion;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Assertions for the field list pattern.
+ */
+class DescriptionListAssert extends BasePatternAssert {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getAssertions(string $variant): array {
+    return [
+      'items' => [
+        [$this, 'assertItems'],
+      ],
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function assertBaseElements(string $html, string $variant): void {
+    $this->assertElementExists('body > .bcl-description-list', new Crawler($html));
```

Comment:
Can we add a check to see that it is the only description list?

Reply:
done.